migrations/sqls/20201028204537-followups-up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS followUps (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  serviceId INT NOT NULL,
+  title VARCHAR(255) NOT NULL,
+  description MEDIUMTEXT,
+  dateOfContact DATE,
```

Comment:
Why is dateOfContact a DATE but appointmentDate is DATETIME?

Reply:
Also, I think `dateOfContact` should be NOT NULL. Appointment date is optional though

Reply:
Oops, that should have stayed DATE, might have been the autofill/smart guess vscode feature.

Reply:
Let's have them both as DATETIME - that way we can capture exact hour of an appointment if needed.